src/routes/+page.ts

```diff
@@ -1,11 +1,35 @@
 import { redirect } from '@sveltejs/kit';
 import { base } from '$app/paths';
 import type { PageLoad } from './$types';
+import { sdk } from '$lib/stores/sdk';
+
+const handleGithubEducationMembership = async (name: string, email: string) => {
+    const result = await sdk.forConsole.billing.setMembership('github-student-developer');
+    if (result && 'error' in result) {
+        await sdk.forConsole.account.deleteSession('current');
+        redirect(
+            303,
+            `${base}/education/error?message=${result.error.message}&code=${result.error.code}`
+        );
+    } else if (result && '$createdAt' in result) {
+        setToGhStudentMailingList(name, email);
+    }
+};
+
+const userVisitedEducationPage = (): boolean => {
+    const didRegisterGithubEducationProgram =
+        localStorage.getItem('githubEducationProgram') === 'true';
+    localStorage.removeItem('githubEducationProgram');
+    return didRegisterGithubEducationProgram;
+};
 
 export const load: PageLoad = async ({ parent, url }) => {
     const { organizations, account } = await parent();
 
-    if (organizations.total) {
+    if (userVisitedEducationPage()) {
+        await handleGithubEducationMembership(account.name, account.email);
+        redirect(303, base);
```

Comment:
is this for the github redirect?

Reply:
On line 7 (https://github.com/appwrite/console/pull/1389/files#diff-436889c325f485f36130aeff9f7f779f6ad37c25881464faf13aecdc2ef75795R7) we do the call to backend to handle the github education membership. If that's all good, we enroll the user to the mailinglist on line 15 (https://github.com/appwrite/console/pull/1389/files#diff-436889c325f485f36130aeff9f7f779f6ad37c25881464faf13aecdc2ef75795R15). If that all went ok, we need to redirect the user to the console (since he's logged in and the organisation (personal team) has been created for them) without going to the usual onboarding flow.